src/BlockTypes/MiniCart.php

```diff
@@ -222,6 +224,39 @@ protected function get_markup() {
 			$cart_contents_total += $cart->get_subtotal_tax();
 		}
 
+		$wrapper_classes = 'wc-block-mini-cart wp-block-woocommerce-mini-cart';
```

Comment:
Because we use `__experimentalSelector`, we don't need `wp-block-woocommerce-mini-cart` here anymore.